salesagility/SuiteCRM
# Next update

## Description




This is an update for the ```next``` branch to bring it in-line with the subtree changes that have been done to Suite-8 legacy internally.

## How To Test This

1. Check Travis
2. Check commit history

## Types of changes


### Final checklist
